app/client/src/widgets/JSONFormWidget/fields/SelectField.test.tsx

```
@@ -70,3 +76,219 @@ describe(".isValid", () => {
+  };
+  let resizeObserver: ResizeObserverMock;
+
+  beforeAll(() => {
+    (
+      global as unknown as { ResizeObserver: typeof ResizeObserverMock }
+    ).ResizeObserver = ResizeObserverMock;
+  });
+
+  afterAll(() => {
+    delete (global as unknown as { ResizeObserver?: typeof ResizeObserverMock })
+      .ResizeObserver;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)